src/swap/v2/router/util.ts

```diff
@@ -0,0 +1,30 @@
+import {SupportedNetwork} from "../../../util/commonTypes";
+import {convertFromBaseUnits} from "../../../util/util";
+import {SwapRoute, SwapRouterQuote} from "../../types";
+import {SWAP_ROUTER_APP_ID} from "./constants";
+
+export function getSwapFeesFromSwapRoute(route: SwapRouterQuote[]): number {
+  return route.reduce((acc, quote) => acc + Number(quote.swap_fees.amount), 0);
```

Comment:
```suggestion
/**
 * @returns The sum of swap fees for the given swap route
 */
export function getTotalSwapFeeForRoute(route: SwapRouterQuote[]): number {
  return route.reduce((totalFee, quote) => totalFee + Number(quote.swap_fees.amount), 0);
```
- Let's add a small description and improve the name
- I think we can use `totalFee` instead of `acc` to make it a bit easier to read

Reply:
I guess we are not using it anywhere so I removed it 👍